trunk/source/RCFGBuilder/src/de/uni_freiburg/informatik/ultimate/plugins/generator/rcfgbuilder/cfg/BoogieIcfgContainer.java

```diff
@@ -89,7 +89,7 @@ public class BoogieIcfgContainer extends ModernAnnotations implements IIcfg<Boog
 	 * Note: This map is only used during construction!
 	 *
 	 */
-	final Map<String, BoogieIcfgLocation> mFinalNode;
+	public final Map<String, BoogieIcfgLocation> mFinalNode;

```

Comment:
I thin so, as this is also used by the IcfgBuilder plugin. Of course, it would be more optimal to move it to some library, but we can do this, if we remove RCFGBuilder in the future.